### PR TITLE
Implement Controller and a Model for DELETE WISHLIST API

### PR DESCRIPTION
**Pull request by sflechas**
```
Name                  Stmts   Miss  Cover
-----------------------------------------
service/__init__.py      22      3    86%
service/models.py        96      5    95%
service/service.py      136      9    93%
-----------------------------------------
TOTAL                   254     17    93%
----------------------------------------------------------------------
Ran 29 tests in 13.449s
```

### CODE REVIEW

File: tests/test_wishlist.py [hunk 1]
```diff
@@ -54,6 +54,17 @@ def tearDown(self):
         db.session.remove()
         db.drop_all()
 
+    def test_delete_wishlist(self):
+        """ Delete a Wishlist """
+        wishlist = Wishlist(name="wishlist_name", customer_id=1234)
+        wishlist_product = WishlistProduct(wishlist_id=123431, product_id=1213321,
```
Comment (nisargthakkar) [inline tests/test_wishlist.py]: I don't think this is needed

Comment (sflechas) [reply tests/test_wishlist.py]: Do you mean the test of the delete?
 Or the product line?  Yes, its needed, I added it to test the cascade delete.

Comment (nisargthakkar) [reply tests/test_wishlist.py]: Okay. That makes sense. But it is not linked to the wishlist created in the previous line. So it won't get deleted. Also, there is no assert to check if the product got deleted or not.

Comment (sflechas) [reply tests/test_wishlist.py]: It had an assert also, doesn't it shows? I'll have to check

Comment (sflechas) [reply tests/test_wishlist.py]: To double check, I temporarily commented out the dropping of the tables and queried them directly.

Comment (nisargthakkar) [reply tests/test_wishlist.py]: Yeah, I think that assert got lost somewhere. Can you break this up into two tests? One where the cascade delete removes a product and one where it doesn't (different wishlist).

Comment (sflechas) [reply tests/test_wishlist.py]: The one to delete the product is part of the other story, this is for wishlists. A wishlist with or without products will be deleted here.

**Review by nisargthakkar — APPROVED** [review]
LGTM

**Pull request merged**